### PR TITLE
Globalization Changes

### PR DESCRIPTION
**Pull request by Bletch1971**
1. Added language selection combobox to the settings form.
2. Added language selection config option.
3. Created a WindowUtils class and added 2 methods to remove the en-US
   default resource dictionary from windows/controls.
4. Added #3 to ALL windows/controls.
5. App.xaml - set default culture to en-US, then load culture value from
   config file.
6. BUGFIX - Added culture convert to NetworkUtils when retrieving version number
   via Json query.

### CODE REVIEW

File: ARK Server Manager/App.xaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:globalizer="clr-namespace:WPFSharp.Globalizer;assembly=WPFSharp.Globalizer"
+             xmlns:globcntrls="clr-namespace:WPFSharp.Globalizer.Controls;assembly=WPFSharp.Globalizer"
```
Comment (ChronosWS) [inline ARK Server Manager/App.xaml]: Several xaml files got this added to them but they do not make any references to this namespace.  They can be removed.

**Comment by ChronosWS** [comment]
I thought the globalizer would manage that resource dictionary automatically once you tell it to change cultures (that's kinda the whole point of the library).  Did you discover this to not be the case?

**Comment by Bletch1971** [comment]
The globalizer does manage the global resource dictionaries, but not the window/control dictionaries. I added the code to remove the window/control dictionaries so it would revert back to the global resource dictionary. You did this in one of the windows, I just added it to the rest and extracted it to a utils class.

If the latest commit, I also found that new resources will return an empty string if not defined in the language file. I have added the en-US language file to the global resource dictionary, so that if the alternate language file (eg de-DE) does not contain the resource, then it reverts back to the en-US value.

**Comment by Bletch1971** [comment]
Cliff, any further changes?

**Pull request merged**